app/models/automation_rule.rb

```diff
@@ -19,29 +19,34 @@
 #
 class AutomationRule < ApplicationRecord
   belongs_to :account
+  has_one_attached :file
 
   validate :json_conditions_format
   validate :json_actions_format
   validates :account_id, presence: true
 
   scope :active, -> { where(active: true) }
 
-  CONDITIONS_ATTRS = %w[country_code status browser_language assignee_id team_id referer].freeze
-  ACTIONS_ATTRS = %w[send_message add_label send_email_to_team assign_team assign_best_agents].freeze
+  CONDITIONS_ATTRS = %w[country_code status message_type browser_language assignee_id team_id referer is_priority_customer company].freeze
```

Comment:
do we have an attribute `is_priority_customer `?

Reply:
It's actually custom_attribute. Need to handle it differently.